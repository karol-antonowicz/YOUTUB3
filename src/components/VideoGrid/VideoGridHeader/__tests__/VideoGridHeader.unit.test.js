import React from 'react';
import {shallow} from 'enzyme';
import {VideoGridHeader} from '../VideoGridHeader';

describe('VideoGridHeader', () => {
    test('renders component with no props', () => {
        const wrapper = shallow(<VideoGridHeader/>);
        expect(wrapper).toMatchSnapshot();
    });
    test('render VideoGridHeader with title props empty string', ()=>{
        const wrapper = shallow(<VideoGridHeader title=''/>);
        expect(wrapper).toMatchSnapshot();
    });
    test('renders VideoGridHeader with dupa title', ()=>{
        const wrapper = shallow(<VideoGridHeader title='duupa & dupa'/>);
        expect(wrapper).toMatchSnapshot();
    });
})