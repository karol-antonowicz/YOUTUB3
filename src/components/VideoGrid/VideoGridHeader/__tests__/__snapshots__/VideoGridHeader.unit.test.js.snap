// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoGridHeader render VideoGridHeader with title props empty string 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;

exports[`VideoGridHeader renders VideoGridHeader with dupa title 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  >
    duupa & dupa
  </span>
</div>
`;

exports[`VideoGridHeader renders component with no props 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;
